### PR TITLE
Coindeblend20

### PR DESCRIPTION
**Pull request by csheneka**
Most important change: 
Added model-file 'seqstack.py' in coindeblend/models, called as 'SeqStack_modular' in analysis/blend2flux.py
Minor changes: 
Added scikit-learn to environment.yml, fixed package name in analysis/-scripts (deblend -> coindeblend), standardised naming datadir in main-files under analysis/

### CODE REVIEW

File: analysis/sextractor_analysis.py [hunk 1]
```diff
@@ -3,9 +3,9 @@
 import pylab as plt
 import pandas as pd
 
-DATADIR = "/Users/alexandre/work/lsst/deblending/coin/candels-blender/output-s_666-n_30000"
-CATDIR = "/Users/alexandre/work/lsst/deblending/coin/candels-blender/output-s_666-n_30000"
-FINALCATDIR = "/Users/alexandre/work/lsst/deblending/coin/catalog_results"
+DATADIR = "<path-to-data-dir>"
+CATDIR = "<path-to-data-dir>"
+FINALCATDIR = "<path-to-catalog-results>"
```
Comment (aboucaud) [inline analysis/sextractor_analysis.py]: thanks ! 😄

**Review by aboucaud — APPROVED** [review]
Thanks a lot @csheneka for fixing some of the mistakes.

File: analysis/sextractor_analysis.py (hunk 1 above)
Comment (aboucaud) [reply analysis/sextractor_analysis.py]: What do you think about using environment variables for this ?

```python
import os 
os.getenv("COINBLEND_DATADIR")
```

Comment (csheneka) [reply analysis/sextractor_analysis.py]: very welcome, thanks to you! 

> What do you think about using environment variables for this ?
> 
> ```python
> import os 
> os.getenv("COINBLEND_DATADIR")
> ```

Right, environment variables would be much more handy, I totally agree!

Comment (aboucaud) [reply analysis/sextractor_analysis.py]: Would you like to do these changes yourself ?

Comment (csheneka) [reply analysis/sextractor_analysis.py]: I can probably do it later today, yes :)

File: analysis/sextractor_analysis.py [hunk 2]
```diff
@@ -1,11 +1,10 @@
+import os
 import sep
 import numpy as np
 import pylab as plt
 import pandas as pd
 
-DATADIR = "<path-to-data-dir>"
-CATDIR = "<path-to-data-dir>"
-FINALCATDIR = "<path-to-catalog-results>"
+CATDIR = "<path-to-catalog-results>"
```
Comment (aboucaud) [inline analysis/sextractor_analysis.py]: Should we do the same with that one ?
Otherwise they need to edit the file.

**Review by aboucaud — COMMENTED** [review]
Changes look good.

Let's also use the same trick for the catalog directory.

File: analysis/sextractor_analysis.py (hunk 1 above)
Comment (csheneka) [reply analysis/sextractor_analysis.py]: Hi, just send another request - the datadir is now set via datadir=os.getenv("COINBLEND_DATADIR") as proposed, and I added a note in the README to export COINBLEND_DATADIR=<path-to-datadir>. There are two issues left in sextractor_analysis.py, the CATDIR in l.7 and l.147 that points to a data file (I'm afraid to touch/delete), and the if{} in l.125-130 that should be obsolete (together with the mix variable).

Comment (aboucaud) [reply analysis/sextractor_analysis.py]: Indeed I saw the obsolete `if` statement. You can remove it as well.

File: analysis/sextractor_analysis.py (hunk 2 above)
Comment (csheneka) [reply analysis/sextractor_analysis.py]: Sure! I wasn't sure about the output structure of candels-blender anymore, but checking on it, seems to be the same datadir for cats?

Comment (csheneka) [reply analysis/sextractor_analysis.py]: I've adjusted it to cat = pd.read_csv(f"{datadir}/{sample}_catalogue.csv"), now the paths/naming should fit together :)

**Review by aboucaud — APPROVED** [review]

**Pull request merged**